docs/users.md

```diff
@@ -280,6 +280,7 @@ clusters registration and support for the [`ManifestWork` API](https://open-clus
 - vcluster: this is based on the [vcluster project](https://www.vcluster.com) and provides the ability to create pods in the hosting namespace of the hosting cluster.
 - host: this control plane type exposes the underlying hosting cluster with the same control plane abstraction
 used by the other control plane types.
+- external: this control plane type represents an existing cluster that was not created or hosted by KubeFlex.
```

Comment:
The "KubeFlex hosting cluster" is _not_ "hosted by KubeFlex". If your intent was to exclude this then I would be clearer on that point. Perhaps something like the following?

> external: this control plane type represents an existing cluster that was not created by KubeFlex and is not the KubeFlex hosting cluster

Reply:
Sounds good to me. Updated.